lib/GameConstants.h

```diff
@@ -567,34 +635,53 @@ namespace MappedKeys
 
 namespace EAiTactic
 {
-enum EAiTactic
-{
-	NONE = -1,
-	RANDOM,
-	WARRIOR,
-	BUILDER,
-	EXPLORER
-};
+	enum EAiTactic
+	{
+		NONE = -1,
+		RANDOM,
+		WARRIOR,
+		BUILDER,
+		EXPLORER
+	};

```

Comment:
ok if it matches current code style, although personally I don't like indentation inside namespace

Reply:
Not sure what to think, personally I am used to c# language where usually everything sits in namespace and namespaced stuff have default intendation

Reply:
Yes I am. Still I am ok if there will be no indentation. Also afaik latest C# introduced a lot of alternative constructs to reduce indentations as well. Like you just write namespace Lalala; and that is all